Mx Platform Security Release Note (#4948)
mendix/docs: content/en/docs/releasenotes/developer-portal/_index.md

## 2022

### August 22nd, 2022

#### New Features

> * Internally we have refactored the location where passwords are stored and validated. This also results in an update to the password reset flow:

Author: Don't think we have to mention the internal work:
```suggestion
* We have changed the password reset flow:
```

>     * The UI on the profile page to update your password has been removed.

Author: ```suggestion
    * You can no longer update your password from your Mendix Profile.
```

Reviewer: "Mendix Profile"

>     * All password resets are now done by visiting the login page and requesting a password reset link in your email inbox.

Author: I think we should mention the **Forgot Password?** link explicitly.
```suggestion
    * All password resets and changes are now done via the [New Password Request page](https://login.mendix.com/mxid3/request-password-reset), which you can reach using the **Forgot password?** link on the login screen. This will send a password reset link to your registered email address.
```

Reviewer: It can mirror the documentation: `You can update your password via the [New Password Request page](https://login.mendix.com/mxid3/request-password-reset), which you can reach using the **Forgot password?** link on the login screen.`

> * The session duration for users logged in with their BYOIDP has been shortened, to give back control to their IDP.

Author: Do we need to mention any specific values here? I can't see anything in our current BYOIDP documentation. Can customers override this?

Reviewer 2: New suggestion:
- We introduced a separate session duration of one hour for users logging into the platform using SSO with their companies BYOIDP, so session management can be better controled by their IDP.

Author: ```suggestion
* We introduced a separate session duration of one hour for users logging into the platform using SSO with their companies BYOIDP, so session management can be better controlled by their IDP.
```

#### Improvements

> * We have made some improvements on the BYOIDP functionality currently in public beta.

Author: Do we need to mention anything explicitly here?
I hope to have some updates to the documentation prepared today, if I have time.

Reviewer: "Public Beta"

Reviewer 2: > Do we need to mention anything explicitly here?
There are no visible user facing changes, but specifically we improved the logic triggered by activating/deactivating an BYOIDP configuration.

Author: I think we should remove this too if the user won't see anything different. Better not to tell the customer about things unless there is a visible/quantifiable difference.

The idea is that this sort of change is recorded internally and doesn't have to be communicated.

> #### Improvements
> 
> * We have made some improvements on the BYOIDP functionality currently in public beta.

Author: Remove this as there are no visible changes.
```suggestion
```

Reviewer 2: Agreed
